src/material.js

```diff
@@ -174,6 +191,19 @@ export class Material {
         return new Lattice(this.lattice);
     }
 
+    getDerivedPropertiesForHash() {
```

Comment:
The name of this function is still incorrect - we never return derivedProperties, only a string for the inchi value